zephyr/esp32s2/CMakeLists.txt

```diff
@@ -213,19 +211,18 @@ if(CONFIG_SOC_SERIES_ESP32S2)
     ../../components/efuse/${CONFIG_SOC_SERIES}/esp_efuse_rtc_calib.c
     ../../components/efuse/${CONFIG_SOC_SERIES}/esp_efuse_rtc_table.c
     ../../components/efuse/${CONFIG_SOC_SERIES}/esp_efuse_utility.c
+    ../../components/soc/${CONFIG_SOC_SERIES}/adc_periph.c
     )
 
   if(CONFIG_PM OR CONFIG_POWEROFF)
     zephyr_sources(
       ../../components/driver/gpio/gpio.c
       ../../components/esp_hw_support/sleep_gpio.c
       ../../components/driver/gpio/rtc_io.c
-      ../../components/esp_hw_support/sleep_modes.c
-      ../../components/esp_hw_support/port/${CONFIG_SOC_SERIES}/rtc_sleep.c
+      # ../../components/esp_hw_support/sleep_modes.c
```

Comment:
should this stay and instead remove L293?
although it is also used for Wi-Fi (check needed)

Reply:
Some of it is also used in v5.1 update.